### PR TITLE
Display HC and forum search results separately 

### PR DESCRIPTION
**Pull request by RobynMartin246**
Here is a bunch of screenshots of what this looks like. 
You can also see it live in sandbox:
- https://gaiagpshelp1556318683.zendesk.com/hc/en-us/

image
Mobile:
image
No Results:
image

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -120,13 +100,31 @@ $(document).ready(function() {
...
+  // Change title of subit request page
+  $("h1:contains('Submit a request')").text("Contact Support");
+  
+  // Capture submit request event
+  $('a.submit-a-request, .article-more-questions a').on('click', function(e) {
+      var path = window.location.pathname;
+      ga('send', 'event', 'Submit Request', 'Submit Request From', path);
+  });
+  
+  $(function() {
```
Comment (RobynMartin246) [inline script.js]: This line and below are new. Everything above is copied from the live HC

**Comment by RobynMartin246** [comment]
@ashlibaldwin this conflict is from the recent New Post>New Feature request change. I will add those changes here before merging if this lgty. (it's live in sandbox if you want to preview it)

**Comment by ashlibaldwin** [comment]
you can merge the other one, then update this PR and I can take another look

**Comment by RobynMartin246** [comment]
@ashlibaldwin all set

**Comment by ashlibaldwin** [comment]
lgtm

**Review by ashlibaldwin — APPROVED** [review]

**Pull request merged**